### PR TITLE
Improved Landing page UI

### LINKED ISSUE #46: Landing Page UI
**Describe the bug** :bug:
The Carousel arrow key is overlapping with the image of the chessboard.
The gradient colour which is used for the card is also too light according to the theme of the landing page

**To Reproduce** :mag:
Steps to reproduce the behaviour:
1. Go to 'https://develop--pbchess.netlify.app/'

**Expected behaviour** :honeybee:
The arrow key and image of the chessboard should not overlap with each other.
The gradient colour should also be a dark colour

**Screenshots** :camera:
original pbChess

**Desktop (please complete the following information):** :computer:
 - OS: Windows10
 - Browser Chrome

### PR DESCRIPTION
**Pull request by Sanchit-Soni**
Fixed Issue: https://github.com/fave77/pbchess/issues/46#issue-830014165

merge

### CODE REVIEW

File: client/src/components/home/home.css [hunk 1]
```diff
@@ -44,20 +44,24 @@ h3 {
   border-radius: 15px;
   color: #ffffff;
   background: rgba(255, 255, 255, 0.5);
-  background: linear-gradient(140deg, rgba(255, 255, 255, 0.5) 0%, rgba(130, 153, 253, 0.5) 100%);
+  background: linear-gradient(
```
Comment (apurv-1) [inline client/src/components/home/home.css:47]: Don't change the gradient as it is design choice.

Comment (apurv-1) [inline client/src/components/home/home.css:47]: Don't change the gradient as it is design choice.

**Review by apurv-1 — CHANGES_REQUESTED** [review]
Undo the linear gradient part and please attach the screenshot.

**Review by apurv-1 — APPROVED** [review]
Looks good to me.

**Review by fave77 — APPROVED** [review]

**Pull request merged**